services/api-server/src/simcore_service_api_server/api/routes/studies_jobs.py

```diff
@@ -81,8 +81,9 @@ async def create_study_job(
     job_inputs: JobInputs,
     webserver_api: Annotated[AuthSession, Depends(get_webserver_session)],
     url_for: Annotated[Callable, Depends(get_reverse_url_mapper)],
+    hidden: Annotated[bool, Query()] = True,
```

Comment:
Add a bit of human-readable doc here :-)  It is not obvious what is a "hidden job"

Reply:
👍